shared/settings/account/index.tsx

```diff
@@ -143,7 +143,7 @@ const AccountSettings = (props: Props) => (
         <Kb.Banner color="yellow" onClose={props.onClearSupersededPhoneNumber}>
           <Kb.BannerParagraph
             bannerColor="yellow"
-            content={`Your unverified phone number ${
+            content={`Your phone number ${
```

Comment:
cc @heronhaye / @zapu I noticed this wording was wrong after having a verified number become superseded. Is that a bug? It makes sense to me that that'd work.